pages/apply/[resident]/summary.tsx

```diff
@@ -81,31 +81,26 @@ const UserSummary = (): JSX.Element => {
     return <Custom404 />;
   }
 
-  const onConfirmData = () => {
+  const onConfirmData = async () => {
```

Comment:
Should the button `await` the call now that it's async, or does that not matter in this particular context?

Reply:
Oh, nvm.
I'm guessing it doesn't because... the button just triggers this background job.
And then if this job succeeds, only then the user gets redirected?